com.geekbench.Geekbench6.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.geekbench.Geekbench6
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: geekbench6
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: geekbench
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - cp -r geekbench/* ${FLATPAK_DEST}/bin/
+      - install -Dm755 geekbench/geekbench6 ${FLATPAK_DEST}/geekbench6
+      - install -Dm644 app-icon.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 com.geekbench.Geekbench6.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml
```

Comment:
I don't see an installed desktop file.

Reply:
I was unsure about that part, due to this being a terminal app and if having a desktop file would mean, it shows in the app drawer

Reply:
you can mark desktop file as being terminal only

without it, no icon.

Reply:
Added a desktop file, but it shows in the app drawer now. Still seems to work fine, starting from there, but you don't get to pass any options.

Reply:
No point in adding a desktop file if users are expected to pass some params, but leaving the decision to you.

Reply:
So, how would I add a desktop file, but keep it from showing up outside of software/the website?

Reply:
`Terminal: true` should hide it from the start menu (or whatever system you DE uses)

Reply:
hrm, I've set that in the .desktop file, as you've likely seen, but it still shows with gnome